### DISCUSSION TITLE
FR: Remember which tab you're editing when switching sites

### DISCUSSION THREAD

**Discussion by riasvdv** [craftcms/cms#8512] [Ideas]
### Description
When switching sites, it would be great to remember which tab you're on when editing an entry.

### Steps to reproduce

1. Edit an entry
2. Switch to a tab
3. Switch to an other site
4. Stay on the same tab

**Comment by arnout-poppr** [comment]
Was this tab-persistence eventually added? Using Craft Pro v4.5.6.1 here but the selected entry tab does not get persisted when changing sites while editing an entry.
